src/stacktrace.cpp

```diff
@@ -5,7 +5,7 @@
 #include <boost/version.hpp>
 
 // only include stacktrace stuff if boost >= 1.65
```

Comment:
i think we can remove the comment here or add "only include stacktrace stuff if boost >= 1.65 **and if OS is not apple**" or something like that.